code-studio/src/js/components/progress/stage_progress.jsx

```diff
@@ -28,7 +28,12 @@ var StageProgress = React.createClass({
       }
 
       var outerClass = (level.kind === 'assessment') ? 'puzzle_outer_assessment' : 'puzzle_outer_level';
-      outerClass = (index === this.props.currentLevelIndex) ? 'puzzle_outer_current' : outerClass;
+      var qtip = "";
+
+      if (index === this.props.currentLevelIndex) {
+        outerClass = 'puzzle_outer_current';
+        qtip = "here is a qtip";
+      }
```

Comment:
Yep! (Let me make sure this is up to date)
On Jun 28, 2016 18:03, "Josh Lory" <notifications@github.com> wrote:

> In code-studio/src/js/components/progress/stage_progress.jsx
> <https://github.com/code-dot-org/code-dot-org/pull/5556#discussion_r68870470>
> :
>
> > @@ -28,7 +28,12 @@ var StageProgress = React.createClass({
> >        }
> >
> >        var outerClass = (level.kind === 'assessment') ? 'puzzle_outer_assessment' : 'puzzle_outer_level';
> > -      outerClass = (index === this.props.currentLevelIndex) ? 'puzzle_outer_current' : outerClass;
> > +      var qtip = "";
> > +
> > +      if (index === this.props.currentLevelIndex) {
> > +        outerClass = 'puzzle_outer_current';
> > +        qtip = "here is a qtip";
> > +      }
>
> I think this can be removed?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/5556/files/e89f193c7c9a3976081629d2b5ac14c68e3d5b11#r68870470>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAeUz8_-YtTKMOLjOQ5ngmkScEvJnzZks5qQcRJgaJpZM4Gmr2B>
> .
>